conan/tools/env/environment.py

```diff
@@ -18,37 +18,43 @@ def environment_wrap_command(env_filenames, env_folder, cmd, subsystem=None,
     if not env_filenames:
         return cmd
     filenames = [env_filenames] if not isinstance(env_filenames, list) else env_filenames
-    bats, shs, ps1s = [], [], []
+    bats, shs, ps1s, fishes = [], [], [], []
 
-    accept = accepted_extensions or ("ps1", "bat", "sh")
+    accept = accepted_extensions or ("ps1", "bat", "fish", "sh")
```

Comment:
grep in the codebase, who is calling ``accepted_extensions=("sh", ))`` and I guess you want to add ``fish`` there too.